mindmeld/cli.py

```diff
@@ -531,6 +535,32 @@ def _get_duckling_pid():
     return pid
 
 
+@shared_cli.command("annotate", context_settings=CONTEXT_SETTINGS)
+@click.option(
+    "--app-path", required=True, help="Needed to locate the application config file.",
+)
+def annotate(app_path):
+    """Runs the annotation command of the Auto Annotator."""
+    importlib.import_module("mindmeld.auto_annotator")
+    config = get_auto_annotator_config(app_path=app_path)
+    annotator = create_annotator(app_path=app_path, config=config)
+    annotator.annotate()
+    logger.info("Annotation Complete.")
+
+
+@shared_cli.command("unannotate", context_settings=CONTEXT_SETTINGS)
+@click.option(
+    "--app-path", required=True, help="Needed to locate the application config file.",
+)
+def unannotate(app_path):
+    """Runs the unannotation command of the Auto Annotator."""
+    importlib.import_module("mindmeld.auto_annotator")
```

Comment:
same

Reply:
Updated to import at the top of the file